### PR TITLE
added code for mjcf postprocessing to make it work better in brax

### LINKED ISSUE #17: move mjcf postprocessing from ksim to sim
Move pawel's script for adding the necessary extra stuff to mjcf artifacts from the sim repo to the ksim repo, and verify that they work

### PR DESCRIPTION
**Pull request by wuallenwu**
based on pawel's code, refactored slightly, add's things like floor, contact, etc. Collision should still be added manually but its very quick

### CODE REVIEW

**Comment by wuallenwu** [comment]
@budzianowski some condim and conaffinity stuff seemed to be a bit wonky for the visual meshes, i updated it now in the code, not sure if this code is being used any where else but worth taking a look at 

Working:
```
visual_geom = ET.Element("default", {"class": "visualgeom"})
        geom_attributes = {"material": "visualgeom", "condim": "3", "contype": "0", "conaffinity": "0"}
        ET.SubElement(visual_geom, "geom", geom_attributes)
 ```
 
 Before:
 ```
visual_geom = ET.Element("default", {"class": "visualgeom"})
        geom_attributes = {"material": "visualgeom", "condim": "1", "contype": "1", "conaffinity": "0"}
        ET.SubElement(visual_geom, "geom", geom_attributes)
 ```

File: ksim/scripts/process_mjcf.py [hunk 1]
```diff
@@ -0,0 +1,456 @@
+# mypy: disable-error-code="operator,union-attr"
+"""Defines common types and functions for exporting MJCF files.
+
+Run:
+    python ksim/scripts/process_mjcf.py /Users/allen/KScale/ksim/ksim/assets/newnew/robot_7dof_arm_merged_simplified.xml
```
Comment (budzianowski) [inline ksim/scripts/process_mjcf.py:5]: remove your directory name

File: ksim/scripts/process_mjcf.py [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+from pathlib import Path
+from typing import List, Union
+
+from kol.formats import mjcf
+
+logger = logging.getLogger(__name__)
+
+STOMPY_HEIGHT = 1.0
+
+DEFAULT_STANDING = {
```
Comment (budzianowski) [inline ksim/scripts/process_mjcf.py:37]: Add todo to move it to a separate definition like in sim

File: ksim/scripts/process_mjcf.py [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+
+        default_standing = DEFAULT_STANDING
+        for joint in joints:
+            if joint.get("name") in default_standing.keys():
+                joint.set("ref", str(default_standing[joint.get("name")]))
+
+        return root
+
+    # def add_joint_limits(self, root: ET.Element, fixed: bool = False) -> None:
+    #     joint_limits = MjcfStompy.default_limits()
```
Comment (budzianowski) [inline ksim/scripts/process_mjcf.py:427]: you can remove it or adapt to use DEFAULT_LIMITS

**Review by budzianowski — APPROVED** [review]
Getting there. Let's actually add the option to add collision addition for specific parts automatically.

**Comment by wuallenwu** [comment]
made all the changes requested - i'll open a separate issue for automatically populating collisions

**Pull request merged**